lasy/utils/laser_utils.py

```diff
@@ -737,9 +738,12 @@ def export_to_z(dim, grid, omega0, z_axis=None, z0=0.0, t0=0.0, backend="NP"):
     time_axis_indx = -1
 
     t_axis = grid.axes[time_axis_indx]
+    if use_cupy:
+        t_axis = xp.asnumpy(t_axis)
```

Comment:
If I don't do this, I later get:
```
lasy/utils/laser_utils.py:744: in export_to_z
    FieldAxprp = ScalarFieldEnvelope(omega0 / c, t_axis)
.../python-3.11/lib/python3.11/site-packages/axiprop/containers.py:101: in __init__
    self.k_freq_base = 2 * np.pi * np.fft.fftfreq(self.Nt, c*self.dt)
.../lib/python3.11/site-packages/numpy/fft/helper.py:169: in fftfreq
    return results * val
cupy/_core/core.pyx:1697: in cupy._core.core._ndarray_base.__array_ufunc__
    ???
cupy/_core/_kernel.pyx:1283: in cupy._core._kernel.ufunc.__call__
    ???
cupy/_core/_kernel.pyx:159: in cupy._core._kernel._preprocess_args
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   TypeError: Unsupported type <class 'numpy.ndarray'>
```

Reply:
`t_axis` is actually not needed on GPU as all propagation in done in the spectral domain -- as soon as we do FFT we don't have time. `self.k_freq_base` can remain on the host